pandas/tests/groupby/test_apply.py

```diff
@@ -1,3 +1,4 @@
+import datetime as dt
```

Comment:
The datetime module is already imported below as well (so you can update your code to use the existing import)